src/applications/disability-benefits/all-claims/pages/contactInformation.js

```diff
@@ -88,8 +79,10 @@ export const uiSchema = {
     emailAddress: emailUI(),
   },
   mailingAddress: {
-    ...addressUISchema(ADDRESS_PATHS.mailingAddress, 'Mailing address', true),
-    'ui:order': uiOrder,
+    ...omit(
+      ['ui:order'],
+      addressUISchema(ADDRESS_PATHS.mailingAddress, 'Mailing address', true),
```

Comment:
I think we can just remove the `ui:order` from the `addressUISchema` since it's specific to form 526 already

`src/applications/disability-benefits/all-claims/utils.jsx` line 371-379

Reply:
There are other pages in the form that import and use that `addressUiSchema` value. I think this `ui:order` was added here so that every page with address fields will have its fields in the same order.